src/main/java/htmlpublisher/HtmlPublisherTarget.java

```diff
@@ -76,6 +76,11 @@
      */
     private final boolean allowMissing;
 
+    /**
+     * If false, several reports with the same name will get published on build page
+     */
+    private final boolean onlyCreateReportWithDifferentName;
```

Comment:
The `private final boolean` way of defining the fields is for the older fields, The newer boolean fields are like the `escapeUnderscores` field and use `private Boolean` for the field (i.e. Boolean object rather than primitive type)

Reply:
corrected